src/components/questions-accordion/QuestionsAccordion.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+
+import ArrowForwardIosSharpIcon from '@mui/icons-material/ArrowForwardIosSharp'
+
+import { AccordionItem } from '~/types'
+import { styles } from '~/components/questions-accordion/QuestionsAccordion.styles'
+
+interface QuestionsAccordion {
+  items: AccordionItem[]
+  title?: string
+  showIcon?: boolean
+}
+
+const QuestionsAccordion: FC<QuestionsAccordion> = ({
+  items,
+  title = '',
+  showIcon = true
+}) => {
+  const [activeQuestions, setActiveQuestions] = useState<number[]>([])
+
+  const onChange = (activeQuestion: number) => {
+    setActiveQuestions((prevActiveQuestions) => {
+      if (prevActiveQuestions.includes(activeQuestion)) {
+        return prevActiveQuestions.filter(
+          (prevActiveQuestion) => prevActiveQuestion !== activeQuestion
+        )
+      } else {
+        return [...prevActiveQuestions, activeQuestion]
+      }
+    })
+  }
+
+  const { t } = useTranslation()
+
+  return (
+    <Box sx={title ? styles.container : {}}>
+      {title ? (

```

Comment:
you can move this outside return and check with &&, so you will not need to use : {} in the end:

const variableName = title && ( 'what you need to render' )